glusterd2/plugin/plugins.go

```diff
@@ -7,13 +7,12 @@ import (
 	"github.com/gluster/glusterd2/plugins/events"
 	"github.com/gluster/glusterd2/plugins/georeplication"
 	"github.com/gluster/glusterd2/plugins/glustershd"
-	"github.com/gluster/glusterd2/plugins/hello"
 	"github.com/gluster/glusterd2/plugins/quota"
 )
 
 // PluginsList is a list of plugins which implements GlusterdPlugin interface
 var PluginsList = []GlusterdPlugin{
-	&hello.Plugin{},
+	// &hello.Plugin{},
```

Comment:
This plugin can be removed completely(Including directory). It was added as example.